### ISSUE TITLE
Maybe something is missing in the configuration file?

### ISSUE THREAD

**Issue by lygamac** [gera2ld/coc-markmap#13]
I copied the exact same configuration from https://github.com/gera2ld/coc-markmap#cocconfig to my coc-settings.json, but texts between `$` are still not rendered as math.

I tried both on my Windows 10 and Archlinux, neither of them worked.

As the default options (`\(a^2\)`, or `$$a^2$$`) is working, my best guess is that the configuration file isn't loaded at all. However, My language servers configurations in coc-settings.json have been loaded.

Is there any dependencies I am missing?

**Comment by lygamac** [comment]
It worked after rebooting the system...

It's worthy to add "maybe you have to restart your computer" as footnote somewhere in the instructions.

**Comment by gera2ld** [comment]
You don't have to restart your computer. It should work after restarting Vim.

**Comment by lygamac** [comment]
@gera2ld Yes and no. When I tried on my second OS, I just pulled my configurations from the previous one. By that means I already have the configuration set up before the extension is installed (or even before Vim has started, but I have to restart Vim to have the installation take effects anyways).

I restarted both Vim and the terminal itself a couple of times and it didn't work out. But after rebooting, the extensions is able to read the configuration and thus just works just by restarting Vim.
